modal/_utils/grpc_utils.py

```diff
@@ -396,7 +397,7 @@ async def _retry_transient_errors(
                 ):
                     last_server_retry_warning_time = now
                     logger.warning(
-                        f"Warning: Received {exc.status} status: {exc.message}. "
+                        f"Warning: Received {exc.status} status: {exc.message}{os.linesep}"
```

Comment:
Can you put the message on a new line too? I think that's usually the most important thing to get in front of user eyes.